js/views/modals/purchase/Purchase.js

```diff
@@ -252,10 +252,18 @@ export default class extends BaseModal {
           contentType: 'application/json',
         })
           .done((data) => {
-            this.updatePageState('pending');
             this.actionBtn.render();
             this.purchase.set(this.purchase.parse(data));
-            this.payment.render();
+            this.payment = this.createChild(Payment, {
```

Comment:
It shouldn't, but in #533 it looked like it was being appended twice. Though that may just be because the pay button could be clicked twice.